hotsos/defs/events/openvswitch/ovs/datapath-checks.yaml

```diff
@@ -9,5 +9,9 @@ port-stats:
 deferred-action-limit-reached:
   input:
     path: 'var/log/kern.log'
-  expr: '(\S+\s+\d+\s+)\S+ .+ (\S+): deferred action limit reached, drop recirc action'
+  # NOTE: the kern.log timestamp pattern is of the form
+  #       "Jun  8 10:48:13 compute4 kernel:"
+  #       or
+  #       "Jun 08 10:48:13 compute4 kernel:"
+  expr: '(\w{3,5}\s+\d{1,2}\s+[\d:]+)\s+.+deferred action limit reached, drop recirc action'
```

Comment:
I have re-checked and need to correct myself slightly. Events do use constraints but they do so by applying them in code rather than using search.constraints in yaml (like in search constraints). So they are at least getting applied. My point about result groups still stands though.